torchtune/modules/model_fusion/fusion_utils.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import functools
+from typing import List
+
+from torch import nn
+
+
+def register_fusion_module(module: nn.Module):
```

Comment:
an example here would be helpful, especially to understand how this is used in combination with the other fusion classes